app/Queries/Feeds/QuestionsFollowingFeed.php

```diff
@@ -27,7 +27,7 @@ public function builder(): Builder
     {
         $followQueryClosure = function (Builder $query): void {
             $query->where('to_id', $this->user->id)
-                ->orWhereIn('to_id', $this->user->following()->select('users.id'));
+                ->orWhereIn('to_id', DB::table('followers')->select('user_id')->where('follower_id', $this->user->id));
```

Comment:
this helps to remove unwanted inner join due to `belongsToMany` relation.

Reply:
I'll suggest ``whereIntegerInRaw`` in please if WhereIn.

![image](https://github.com/user-attachments/assets/43951a52-8c77-41b6-9e57-6cc87d5d3ded)

Reply:
There are no bindings 🤞